src/encoded/static/components/filegallery.js

```diff
@@ -2815,6 +2815,20 @@ const FileDetailView = function FileDetailView(node, qcClick, auditIndicators, a
                         </div>
                     : null}
 
+                    {selectedFile.cropped_read_length !== undefined ?
+                        <div data-test="croppedreadlength">
+                            <dt>Cropped read length</dt>
+                            <dd>{selectedFile.cropped_read_length}</dd>
+                        </div>
+                    : null}
+
+                    {selectedFile.cropped_read_length_tolerance !== undefined ?
```

Comment:
Why are you checking specifically for undefined? Why not:

{selectedFile.ropped_read_length_tolerance ?

Reply:
I am checking to see if cropped_read_length_tolerance has a value to be displayed.